### ISSUE TITLE
[BUG] management command "cms check" doesn't work

### ISSUE THREAD

**Issue by MichaelAnckaert** [django-cms/django-cms#7033]
## Description

When running the command `python manage.py cms check` on the develop branch there is an issue with the argument parsing of the custom management command. 

I am aware that this happens on a development branch. This issue is mainly to raise awareness and coordinate any efforts to fix this issue. 

## Steps to reproduce

Run the command  `python manage.py cms check` and the following output is logged:

> 
> Traceback (most recent call last):
>   File "/home/michael/projects/django-cms-demo/myproject/manage.py", line 22, in <module>
>     main()
>   File "/home/michael/projects/django-cms-demo/myproject/manage.py", line 18, in main
>     execute_from_command_line(sys.argv)
>   File "/home/michael/projects/django-cms-demo/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
>     utility.execute()
>   File "/home/michael/projects/django-cms-demo/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
>     self.fetch_command(subcommand).run_from_argv(self.argv)
>   File "/home/michael/projects/django-cms-demo/venv/lib/python3.9/site-packages/django/core/management/base.py", line 355, in run_from_argv
>     self.execute(*args, **cmd_options)
>   File "/home/michael/projects/django-cms-demo/venv/lib/python3.9/site-packages/django/core/management/base.py", line 391, in execute
>     if self.requires_system_checks and not options['skip_checks']:
> KeyError: 'skip_checks'
> 

## Expected behaviour

CMS checks are performed without error. 

## Actual behaviour

The parsing of the management command fails with an error on parsing the arguments passed to the management command.

## Additional information (CMS/Python/Django versions)

Regular management commands (for example `manage.py check`) work without issue. 

* Django CMS: develop branch
* Python 3.9
* Installed packages:

asgiref==3.3.4
Django==3.2.2
django-classy-tags==1.0.0
django-cms @ file:///home/michael/projects/django-cms
django-formtools==2.2
django-sekizai==2.0.0
django-treebeard==4.3.1
djangocms-admin-style==2.0.2
pytz==2021.1
six==1.16.0
sqlparse==0.4.1

**Comment by goutnet** [comment]
we will need to check this on the coming release 3.9.0 and fix it before release then.

@vinitkumar  @marksweb  could you help me check?

**Comment by marksweb** [comment]
@goutnet absolutely. Only just seen this so will try to take a look tonight when I'm doing some work 👍

**Comment by goutnet** [comment]
now that 3.9.0rc1 is out, can we check this on that release?

**Comment by marksweb** [comment]
@goutnet I'm not seeing this with 3.9rc1

I get the following;

```
********************************
Checking django CMS installation
********************************

Sekizai
=======

  - Sekizai is installed [OK]
  - Sekizai template context processor is installed [OK]
  - Sekizai namespaces 'js' and 'css' found in 'page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'feature.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/australia/article.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/australia/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/australia/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/australia/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/bath-half/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/bath-half/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/bchc/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/bchc/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/brainathlon/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/brainathlon/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/brainathlon/content.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/brainathlon/content-entered.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/bwhc/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/bwhc/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/get-britain-running/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/get-britain-running/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/get-britain-running/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/get-britain-running/petition.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/hapalua/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/hapalua/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/hapalua/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/honolulu/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/honolulu/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/honolulu/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/ireland/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/ireland/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/ireland/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/kidney-care/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/kidney-care/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/napoli/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/napoli/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/napoli/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/roh/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/roh/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/rourke-cycles/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/rourke-cycles/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/run-czech_prague-marathon/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/run-czech_prague-marathon/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/run-czech_prague-marathon/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/run-czech_woman-challenge/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/run-czech_woman-challenge/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/run-czech_woman-challenge/page.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/sorrento/festival.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/sorrento/home.html' [OK]
  - Sekizai namespaces 'js' and 'css' found in 'themes/sorrento/page.html' [OK]

Sekizai configuration okay [OK]

Internationalization
====================

  - New style CMS_LANGUAGES [OK]

Middlewares
===========

  - django.contrib.sessions.middleware.SessionMiddleware middleware must be in MIDDLEWARE [ERROR]

Context processors
==================


Plugin instances
================

  - Plugin instances of 7 types found in the database [OK]

The plugins in your database are in good order [OK]

Presence of "copy_relations"
============================


All plugins and page/title extensions have "copy_relations" method if needed. [OK]

PlaceholderField
================


PlaceholderField configuration okay [OK]


OVERALL RESULTS
===============

1 errors!
55 checks successful!

Please check the errors above

CommandError: 
(.venv) mwalker@CH-Mac075  % 
```

Not sure what that `CommandError` is yet, and I'm reminded that the middleware check annoys me because I've a subclass of `SessionMiddleware`

**Comment by mbi** [comment]
Confirmed, I'm still seeing this in 3.9.0rc1 installed from the updated zip from #7048 
``` 
(.venv)  ✘  ~/Code/living-archives/living_archives   upgrade_32_cms39  python manage.py cms check 
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/home/marco/Code/living-archives/.venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/marco/Code/living-archives/.venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/marco/Code/living-archives/.venv/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/marco/Code/living-archives/.venv/lib/python3.8/site-packages/django/core/management/base.py", line 391, in execute
    if self.requires_system_checks and not options['skip_checks']:
KeyError: 'skip_checks'
```

**Comment by MichaelAnckaert** [comment]
This issue still occurs when testing on the `develop` branch. Haven't had time to test it out with the 3.9.0rc1 but I believe this should be the same result?

**Comment by goutnet** [comment]
can you share a bit more of your setup to see how we can reproduce this issue?

3.9.0rc1 and develop are very close
3.9.0rc2 (just released) only fixes the assets.

**Comment by mbi** [comment]
@goutnet if it helps, here is a very minimal test case based on our [project template](https://github.com/cruncher/project-template).

```bash
python3 -m venv .venv
source .venv/bin/activate
python3 --version
# Python 3.8.10

pip install Django==3.2.3

$VIRTUAL_ENV/bin/django-admin startproject --template=https://github.com/cruncher/project-template/zipball/master issue7033
cd issue7033
pip install -r requirements.txt
cd issue7033
python manage.py cms check

# ... KeyError: 'skip_checks'
```

**Comment by goutnet** [comment]
using this scenario I am able to reproduce your issue, thanks for sharing, but by the look of it the execution breaks in django code (before hitting CMS). Let me check this.

**Comment by goutnet** [comment]
After upgrading treebeard to 4.5.1 seems the issue is gone, can you check on your end?

`pip install --upgrade  django-treebeard`

**Comment by goutnet** [comment]
@MichaelAnckaert @mbi can you check if upgrading treebeard in your installation fixes the issue?

**Comment by marksweb** [comment]
I have `django-treebeard==4.4` with my working example from above.

**Comment by mbi** [comment]
@goutnet 

Not related, apparently:

```bash 
pip install --upgrade django-treebeard==4.5.1
pip freeze | grep beard
# django-treebeard==4.5.1
python manage.py cms check   
# ...KeyError: 'skip_checks'
```

**Comment by mbi** [comment]
@goutnet here is the bug:

https://github.com/django-cms/django-cms/blob/develop/cms/management/commands/subcommands/base.py#L40

If you also import `DJANGO_3_2`  [here](https://github.com/django-cms/django-cms/blob/develop/cms/management/commands/subcommands/base.py#L8) and then include `or DJANGO_3_2` in the test [here](https://github.com/django-cms/django-cms/blob/develop/cms/management/commands/subcommands/base.py#L40), the problem is fixed.

**Comment by MichaelAnckaert** [comment]
Can confirm that the fix by @mbi (#7054 ) fixes the issue. 

Updating the `django-treebeard` package had no effect.

**Comment by marksweb** [comment]
Alright, I've been working on too many projects & too many upgrades to 3.2 😆 

I've confirmed this bug, tested the PR and can confirm the first 👍

**Comment by goutnet** [comment]
Fixed onto `release/3.9.x` branch, thanks!

we will merge `release/3.9.x` onto develop when the release gets stable :)
